tests/clustering/test_preprocessor.py

```diff
@@ -0,0 +1,43 @@
+from urbansearch.clustering import preprocessing
+p = preprocessing.PreProcessor()
+
+
+def test_stem():
+    text = ["handel", "bemoederen", "koe"]
+    expected = ["handel", "bemoeder", "koe"]
+    result = p.stem(text)
+    for w in result:
+        print(w)
+    assert (result == expected)
+
+
+def test_strip():
```

Comment:
More elaborate testing should be done for the stripping. Think about overlapping occurrences etc